l10n_pt_account_asset/data/account_legal_rate.xml

```diff
@@ -0,0 +1,333 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
```

Comment:
If this is an official code table, we might prefer to not have `noupdate="1"`, so that later versions can modify older entries in the table.